tools/deploy

```diff
@@ -0,0 +1,21 @@
+#! /usr/bin/bash
+
+deployment_name=$1
+account_id=$2
+secrets_yaml=$3
+environment=$4
+
+# clear out and update the helm charts
+rm -r hub/charts/*
+helm dependency update hub
+
+tmp_decrypted=~/staging.yaml.decrypted
+
+# have to be in the same directory as .sops.yaml
+pushd $(dirname ${secrets_yaml})
+sops --decrypt $(basename ${secrets_yaml}) > $tmp_decrypted
+
+popd
+helm upgrade --wait --install --namespace ${deployment_name}-${environment} ${deployment_name}-${environment} hub -f deployments/${deployment_name}/config/common.yaml -f deployments/${deployment_name}/config/${environment}.yaml -f ${tmp_decrypted} --set-string jupyterhub.singleuser.image.name=${account_id}.dkr.ecr.us-east-1.amazonaws.com/${deployment_name}-user-image
```

Comment:
can we drop that --set-string since it's now included in the common.yaml?

Reply:
can we get a #TODO get sops to work in-place if possible?

Reply:
what do you mean by "work in-place"?